src/process.rs

```diff
@@ -652,6 +655,35 @@ impl Native {
         wait_forever().await
     }
 
+    #[cfg(windows)]
+    async fn signal_loop_tokio(&self, _: u32) -> io::Result<()> {
+        // on windows Ctrl+C signals are propagated automatically and no other
+        // signals are supported, so there is nothing to do here
+        wait_forever().await;
+        Ok(())
+    }
+
+    #[cfg(unix)]
+    async fn signal_loop_tokio(&self, pid: u32) -> io::Result<()> {
```

Comment:
Huh, I'm surprised that this is necessary. SIGINT gets sent to everything that is in the process group that owns the terminal, if I remember correctly; do the children get spawned in other process groups, I guess?